### ISSUE TITLE
Online Live SLD Editor

### ISSUE THREAD

**Issue by 1Map** [NieuwlandGeo/SLDReader#22]
Nice component.  It would now be nice to have a live online SLD Editor where you preview your changes on the map as you edit the style.  I mean editing the sld in a user friendly way (not just the raw sld file).  I tried something some time back using:

https://github.com/highsource/ogc-schemas

especially with:

https://github.com/highsource/ogc-schemas/blob/master/scripts/lib/SLD_1_0_0_GeoServer.js

To build a style editor with rules and all that can be used in geoserver (Or with your component).

Basically the script above will transform a raw SLD to Javascript Objects, which can in turn be used again to fill your interface with the values.  The good thing here is that you can the give back the Javascript Objects and it will transform it back to a SLD string (So it works bi-directional).
